### ISSUE TITLE
What is the meaning of "Received UpdateClientMessage for unknown player"?

### ISSUE THREAD

**Issue by parautenbach** [postlund/pyatv#901]
What do these messages mean? 

Can one interact with these "players" in some way?

```
2020-12-01 21:51:46 WARNING (MainThread) [pyatv.mrp.player_state] Received UpdateClientMessage for unknown player com.apple.TVWatchList
2020-12-01 21:51:46 WARNING (MainThread) [pyatv.mrp.player_state] Received UpdateClientMessage for unknown player com.nousguide.rbms
2020-12-01 21:51:47 WARNING (MainThread) [pyatv.mrp.player_state] Received UpdateClientMessage for unknown player com.plexapp.plex
2020-12-01 21:51:47 WARNING (MainThread) [pyatv.mrp.player_state] Received UpdateClientMessage for unknown player com.showmax.main
2020-12-01 21:51:47 WARNING (MainThread) [pyatv.mrp.player_state] Received UpdateClientMessage for unknown player com.netflix.Netflix
2020-12-01 21:51:47 WARNING (MainThread) [pyatv.mrp.player_state] Received UpdateClientMessage for unknown player com.apple.TVAirPlay
```

(I've spent some time searching and got very few hits.)

**Comment by postlund** [comment]
A player is normally something that plays media, e.g. an app. MRP has a structure to represent who is playing what and where, mostly to support various use cases. I can't say that I have full knowledge of how it works.

Anyways... the `UpdateClientMessage` (defined [here](https://github.com/postlund/pyatv/blob/master/pyatv/mrp/protobuf/UpdateClientMessage.proto)) is used by the device to change some metadata about a player. It's not very interesting per se, I only use it for display name:

```
message NowPlayingClient {
  optional int32 processIdentifier = 1;
  optional string bundleIdentifier = 2;
  optional string parentApplicationBundleIdentifier = 3;
  optional int32 processUserIdentifier = 4;
  optional int32 nowPlayingVisibility = 5;
//    optional TintColor tintColor = 6;
  optional string displayName = 7;
  repeated string bundleIdentifierHierarchys = 8;
}
```

Previously I stored the client information when a `SetStateMessage` was sent (which includes a `playerPath`, which in turn contains a `client`), but in tvOS 14 Apple decided to start sending update messages prior to including it in the `SetStateMessage` which broke my logic. It was fixed in #880 that was included in the 0.7.5 release.
